examples/cloud-tomee-azure/pom.xml

```diff
@@ -0,0 +1,128 @@
+                <version>${project.version}</version>
+                <configuration>
+                    <tomeeClassifier>webprofile</tomeeClassifier>
+                    <!--You can tweak this for your case-->
+                    <args>-Xmx512m -XX:PermSize=256m</args>
+                    <!--Azure requires this port and the default on TomEE is 8080-->
+                    <tomeeHttpPort>80</tomeeHttpPort>
+                    <!--You can place here, if needed, all the config files needed for the tomee/conf folder-->
+                    <!--<config>${project.basedir}/src/main/tomee/</config>-->
+                </configuration>
+                <executions>
+                    <!--This will create a TomEE embedded executable jar-->
+                    <execution>
+                        <id>executable-jar</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <!--This creates the exec suffix in the embedded TomEE fat jar-->
+                            <classifier>exec</classifier>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>azure-single-jar</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.microsoft.azure</groupId>
+                        <artifactId>azure-webapp-maven-plugin</artifactId>
+                        <!-- check Maven Central for the latest version -->
+                        <version>1.5.2</version>
+                        <executions>
+                            <execution>
+                                <!-- This needs a working azure account and would break the build. Use explicitly with: -->
+                                <!-- mvn -Pazure azure-webapp:deploy -->
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <authentication>
+                                <!-- If you invoke the plugin from a console with an Azure CLI with -->
+                                <!-- authenticated session, you don't need this block. -->
+                                <serverId>azure-auth</serverId>
+                            </authentication>
+                            <deploymentType>jar</deploymentType>
+                            <jarFile>${project.build.directory}/${project.build.finalName}-exec.jar</jarFile>
+                            <stopAppDuringDeployment>true</stopAppDuringDeployment>
+                            <resourceGroup>tomee-group</resourceGroup>
+                            <appName>cloud-tomee-azure</appName>
```

Comment:
@brunobat I found the issue. Is in this line. The app name must be unique because that will conform the subdomain:
https://<appNameValueHere>.azurewebsites.net/

I think the examples needs to add a step for changing this name and the default value should be something like:
```
<appName>Dont-Foret-To-Change</appName>
```
And provide the example of the error one will see if they try to use an existing name.

Reply:
Got it, changed the pom file